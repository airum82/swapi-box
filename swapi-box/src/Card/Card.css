.card {
  padding-bottom: 4%; }

.card button {
  color: #7a2e2e;
  background-color: #baa948;
  padding: 5%;
  font-size: 1em;
  border: none; }
