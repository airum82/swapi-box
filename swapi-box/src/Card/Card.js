import React from 'react';

export const Card = (props) => {
  return (
    <div>
      <h2>name</h2>
      <ul>
        <li>Homeworld</li>
        <li>Species</li>
        <li>Language</li>
        <li>Population of Homeworld</li>
      </ul>
      <button>Favorite</button>
    </div>

  )
}