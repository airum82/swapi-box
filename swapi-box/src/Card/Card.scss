.card {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-bottom: 4%;
}

.card h2 {
  font-size: 1.5em;
}

.card button {
  color: #7a2e2e;
  background-color: #baa948;
  padding: 5%;
  font-size: 1em;
  border: none;
}
