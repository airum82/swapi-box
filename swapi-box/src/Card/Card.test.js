import React from 'react';
import { shallow } from 'enzyme';
import { Card } from './Card';

describe('Card component', () => {
  it('Should match snapshot upon render', () => {
    const wrapper = shallow(<Card />);

    expect(wrapper).toMatchSnapshot();
  })
})