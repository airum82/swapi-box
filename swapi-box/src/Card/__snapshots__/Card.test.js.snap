// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component Should match snapshot upon render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          name
</h2>,
        <ul>
          <li>
                    Homeworld
          </li>
          <li>
                    Species
          </li>
          <li>
                    Language
          </li>
          <li>
                    Population of Homeworld
          </li>
</ul>,
        <button>
          Favorite
</button>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "name",
        },
        "ref": null,
        "rendered": "name",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              Homeworld
</li>,
            <li>
              Species
</li>,
            <li>
              Language
</li>,
            <li>
              Population of Homeworld
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Homeworld",
            },
            "ref": null,
            "rendered": "Homeworld",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Species",
            },
            "ref": null,
            "rendered": "Species",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Language",
            },
            "ref": null,
            "rendered": "Language",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Population of Homeworld",
            },
            "ref": null,
            "rendered": "Population of Homeworld",
            "type": "li",
          },
        ],
        "type": "ul",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Favorite",
        },
        "ref": null,
        "rendered": "Favorite",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            name
</h2>,
          <ul>
            <li>
                        Homeworld
            </li>
            <li>
                        Species
            </li>
            <li>
                        Language
            </li>
            <li>
                        Population of Homeworld
            </li>
</ul>,
          <button>
            Favorite
</button>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "name",
          },
          "ref": null,
          "rendered": "name",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                Homeworld
</li>,
              <li>
                Species
</li>,
              <li>
                Language
</li>,
              <li>
                Population of Homeworld
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Homeworld",
              },
              "ref": null,
              "rendered": "Homeworld",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Species",
              },
              "ref": null,
              "rendered": "Species",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Language",
              },
              "ref": null,
              "rendered": "Language",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Population of Homeworld",
              },
              "ref": null,
              "rendered": "Population of Homeworld",
              "type": "li",
            },
          ],
          "type": "ul",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Favorite",
          },
          "ref": null,
          "rendered": "Favorite",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
